R/geom-bar.r

```diff
@@ -123,16 +138,18 @@ GeomBar <- ggproto("GeomBar", GeomRect,
     params
   },
 
-  extra_params = c("na.rm", "orientation"),
+  extra_params = c("just", "na.rm", "orientation"),
 
   setup_data = function(data, params) {
     data$flipped_aes <- params$flipped_aes
     data <- flip_data(data, params$flipped_aes)
     data$width <- data$width %||%
       params$width %||% (resolution(data$x, FALSE) * 0.9)
+    data$just <- params$just %||% 0.5
     data <- transform(data,
       ymin = pmin(y, 0), ymax = pmax(y, 0),
-      xmin = x - width / 2, xmax = x + width / 2, width = NULL
+      xmin = x - width * (1 - just), xmax = x + width * just,
+      width = NULL, just = NULL
```

Comment:
```suggestion
      width = NULL
```

This `width = NULL` is to remove `width` column (if exists) passed from the `Stat`. As the data doesn't have `just` column, we don't need this `just = NULL`.